### PR TITLE
feat: adding Social Sign in to price plan

### PR DESCRIPTION
**Pull request by BlueAcidFrog**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/page-content/pricing-tiers/pricing-cloud.tsx [hunk 1]
```diff
@@ -89,6 +89,10 @@ export const pricingCloud: Pick<PricingProps, 'cloud'> = {
           icon: Plus,
           title: 'Unlimited identities'
         },
+        {
+          icon: Plus,
+          title: 'Social Sign-in'
```
Comment (aeneasr) [inline src/page-content/pricing-tiers/pricing-cloud.tsx:94]: This is also avialable on developer plan :)

Comment (BlueAcidFrog) [reply src/page-content/pricing-tiers/pricing-cloud.tsx:98]: done, ready to merge :)

**Review by vinckr — COMMENTED** [review]
Technically you could remove "Social SignIn" from the Start-Up plan, since it says:

> All Developer features, plus

And social is part of developer features.

**Comment by BlueAcidFrog** [comment]
@vinckr true, you wanna remove it?

File: src/page-content/pricing-tiers/pricing-cloud.tsx [hunk 2]
```diff
@@ -89,6 +93,10 @@ export const pricingCloud: Pick<PricingProps, 'cloud'> = {
...
+        },
```
Comment (vinckr) [inline src/page-content/pricing-tiers/pricing-cloud.tsx:99]:
```suggestion
```
as suggested here: https://github.com/ory/web/pull/604#pullrequestreview-925741982

**Pull request merged**